FusionIIIT/applications/iwdModuleV2/migrations/0001_initial.py

```diff
@@ -144,58 +65,154 @@ class Migration(migrations.Migration):
+                ('periodOfHindrance', models.IntegerField()),
+                ('periodOfExtension', models.IntegerField()),
+                ('key', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='iwdModuleV2.projects')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='CorrigendumTable',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('issueDate', models.DateField()),
+                ('nitNo', models.IntegerField()),
                 ('name', models.CharField(max_length=200)),
-                ('description', models.CharField(max_length=200)),
+                ('lastDate', models.DateField(null=True)),
+                ('lastTime', models.TimeField()),
+                ('env1BidOpeningDate', models.DateField()),
+                ('env1BidOpeningTime', models.TimeField()),
+                ('env2BidOpeningDate', models.DateField()),
+                ('env2BidOpeningTime', models.TimeField()),
+                ('key', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='iwdModuleV2.projects')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Agreement',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('date', models.DateField()),
+                ('agencyName', models.CharField(max_length=200)),
+                ('workName', models.CharField(max_length=200)),
+                ('fdrSum', models.IntegerField()),
+                ('key', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='iwdModuleV2.projects')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='AESDetails',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('sNo', models.CharField(max_length=100)),
+                ('descOfItems', models.CharField(max_length=200)),
+                ('unit', models.CharField(max_length=200)),
+                ('quantity', models.IntegerField()),
+                ('rate', models.IntegerField()),
+                ('amount', models.IntegerField()),
+                ('key', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='iwdModuleV2.projects')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Addendum',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('issueDate', models.DateField()),
+                ('nitNiqNo', models.IntegerField()),
+                ('name', models.CharField(max_length=200)),
+                ('openDate', models.DateField()),
+                ('openTime', models.TimeField()),
```

Comment:
Remove migrations from the PR

Reply:
Done

Reply:
Please do check that deleting init.py also does not create any problem.